packages/components/src/Account/Auth/EmailAuth.tsx

```diff
@@ -30,68 +34,129 @@ export interface AccountEmailAuthProps {
 
 export interface AccountEmailAuthState {
   emailAddress: string;
-  errorMessage: string | null;
+  errorMessage: string | undefined;
+  hasAgreedToTOS: boolean;
+  hasSelectedToAddToNewsletter: boolean;
+  hasBlurred: boolean;
+}
+
+// TODO(jorgelo): Is there a more official way to do this?
+
+function isValidEmail(email: string): boolean {
```

Comment:
You should add `isValidEmail ` as a util if there isn't one already. Also there is some regex in utils you can either add the email or use what's there if already available.

Reply:
Done, created it in utils.